### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 1]
```diff
@@ -15,6 +15,9 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return (result1 - bet1 == 0 && result2 - bet2 == 0)
+                ? 2 : ((result1 < result2 && bet1 < bet2)
+                ? 1 : ((result1 > result2 && bet1 > bet2)
+                ? 1 : ((result1 == result2 && bet1 == bet2) ? 1 : 0)));
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: too many conditions '?' try to simplify it

Comment (ghost) [reply src/main/java/core/basesyntax/FootballMatch.java]: done

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 2]
```diff
@@ -15,6 +15,11 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        Integer resultOne = result1;
+        Integer resultTwo = result2;
+        Integer betOne = bet1;
+        Integer betTwo = bet2;
+        return (result1 - bet1 == 0 && result2 - bet2 == 0)
+                ? 2 : (resultOne.compareTo(resultTwo) == (betOne.compareTo(betTwo)) ? 1 : 0);
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: try to use it without redundant variables https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html

Comment (ghost) [reply src/main/java/core/basesyntax/FootballMatch.java]: done

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 3]
```diff
@@ -15,6 +15,8 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return result1 == bet1 && result2 == bet2 ? 2 : result1 < result2 && bet1 < bet2
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: '?' should be on a new line

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 4]
```diff
@@ -15,6 +15,8 @@
...
+                || result1 > result2 && bet1 > bet2
+                || result1 == result2 && bet1 == bet2 ? 1 : 0;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: same

Comment (ghost) [reply src/main/java/core/basesyntax/FootballMatch.java]: done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**